### PR TITLE
A few small examples I've been meaning to clean up 

### PR DESCRIPTION
**Pull request by septract**
* A couple of small examples of CN syntax 
* Some small error cases I found

### CODE REVIEW

**Comment by dc-mak** [comment]
Some of these have links to issues attached and some of these don't. Is that intentional? I'm fine to let it be like this, but just wanted to check.

File: src/example-archive/simple-examples/inprogress/int_queue_pop_mdd.c [hunk 1]
```diff
@@ -0,0 +1,83 @@
+// Variant on the queue example in src/examples. Currently doesn't work. 
+
+#include "list.h"
+#include "list_snoc_spec.h"
```
Comment (dc-mak) [inline src/example-archive/simple-examples/inprogress/int_queue_pop_mdd.c:4]: If you're running within the same directory, it should be:
```
#include "../../../examples/list.h"
#include "../../../examples/list_snoc.h"
```
Alternatively, copy over the headers into here if you want?

What's the purpose of this example? Is it a bad error message or something else?

**Review by dc-mak — APPROVED** [review]
Approving to not delay by another day but would still like clarifications requested.

File: src/example-archive/simple-examples/inprogress/int_queue_pop_mdd.c (hunk 1 above)
Comment (septract) [reply src/example-archive/simple-examples/inprogress/int_queue_pop_mdd.c:4]: I think this is redundant now @bcpierce00 has finished his queue example. I'll delete it.

**Pull request merged**